xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -123,14 +128,44 @@ bool CVAAPIContext::EnsureContext(CVAAPIContext **ctx, CDecoder *decoder)
   return true;
 }
 
+namespace
+{
+int find_open_render_node_fd()
```

Comment:
You can see that in e.g. Android: https://cgit.freedesktop.org/vaapi/libva/tree/va/android/va_android.cpp

The only method actually using this dependency is: vaPutSurface - nothing else has a dependency on the display server.